src/path.js

```diff
@@ -16,6 +16,285 @@ function Path() {
     this.strokeWidth = 1;
 }
 
+function roundDecimal(float, places) {
+    return +(Math.round(float + 'e+' + places) + 'e-' + places);
+}
+
+function optimizeCommands(commands) {
+    // separate subpaths
+    let subpaths = [[]];
+    for (let i = 0; i < commands.length; i += 1) {
+        const subpath = subpaths[subpaths.length - 1];
+        const cmd = commands[i];
+        const firstCommand = subpath[0];
+        const secondCommand = subpath[1];
+        const previousCommand = subpath[subpath.length - 1];
+        subpath.push(cmd);
+        if (cmd.type === 'Z') {
+            // When closing at the same position as the path started,
+            // remove unnecessary line command
+            if (
+                firstCommand.type === 'M' &&
+                secondCommand.type === 'L' &&
+                previousCommand.type === 'L' &&
+                previousCommand.x === firstCommand.x &&
+                previousCommand.y === firstCommand.y
+            ) {
+                subpath.shift();
+                subpath[0].type = 'M';
+            }
+
+            if (i + 1 < commands.length) {
+                subpaths.push([]);
+            }
+        } else if (cmd.type === 'L') {
+            // remove lines that lead to the same position as the previous command
+            if (previousCommand.x === cmd.x && previousCommand.y === cmd.y) {
+                subpath.pop();
+            }
+        }
+    }
+    commands = [].concat.apply([], subpaths); // flatten again
+    return commands;
+}
+
+/**
+ * Returns options merged with the default options for parsing SVG data
+ * @param {object} options (optional)
+ */
+function createSVGParsingOptions(options) {
+    const defaultOptions = {
+        decimalPlaces: 2,
+        optimize: true,
+        flipY: true,
+        scale: 1,
+        x: 0,
+        y: 0
+    };
+    const newOptions = Object.assign({}, defaultOptions, options);
+    return newOptions;
+}
+
+/**
+ * Returns options merged with the default options for outputting SVG data
+ * @param {object} options (optional)
+ */
+function createSVGOutputOptions(options) {
+    // accept number for backwards compatibility
+    // and in that case set flipY to false
+    if (parseInt(options) === options) {
+        options = { decimalPlaces: options, flipY: false };
+    }
+    const defaultOptions = {
+        decimalPlaces: 2,
+        optimize: true,
+        flipY: true
```

Comment:
we shall keep it `false`, and flip it back to `true` in 2.0.0
so we could merge this PR now

Reply:
Is there a plan to release a 1.x version before 2.0.0? Because we already have the breaking change regarding the platform-agnostic `font.table.name` values merged in our main branch

Reply:
We need to clarify this road map (with @fdb ?)
If the next versions is going to be 2.0.0(-alpha1) then no problem for me